server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -253,6 +255,26 @@ public void start() throws Exception
           @Override
           public void stop()
           {
+            if (activeConnections.get() != 0) {
```

Comment:
I think ```activeConnections.get() > 0``` is more appropriate because we don't have to wait even if it returns a negative value due to some bugs.